.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,17 @@
+<!-- Please take a look at our [Contributing](../blob/master/CONTRIBUTING.md)
+documentation before submitting a Pull Request!
+Thank you for contributing to Crossplane! -->
+
+**Description of your changes:**
+
+**Which issue is resolved by this Pull Request:**
+Resolves #
+
+**Checklist:**
+- [ ] Documentation has been updated, if necessary.
+- [ ] Pending release notes updated with breaking and/or notable changes, if necessary.
+- [ ] Upgrade from previous release is tested and upgrade user guide is updated, if necessary.
+- [ ] Code generation (`make generate`) has been run to update object specifications, if necessary.
+- [ ] CRD manifests generation (`make manifests`) has been run to update CRD manifests yaml file specifications, if necessary.
+- [ ] Update dependencies (`make vendor`) has been run to update `Gopkg.lock` file, if necessary
+- [ ] Comments have been added or updated based on the standards set in [CONTRIBUTING.md](../blob/master/CONTRIBUTING.md#comments)
```

Comment:
I think we should add `Commits have been squashed to improve readability`

Reply:
Fixed